terraform/github/main.tf

```
@@ -329,3 +342,11 @@ resource "github_team_repository" "modernisation-platform-environments-access" {
   repository = module.modernisation-platform-environments.repository.id
   permission = "push"
 }
+
+# Give write access to teams on the configuration management repo (access to merge to main is restricted by codeowners file)
+resource "github_team_repository" "modernisation-platform-configuration-management-access" {
+  for_each   = { for team in local.application_teams : team => team }
+  team_id    = each.value
+  repository = module.modernisation-platform-environments.repository.id
```

```suggestion
  repository = module.modernisation-platform-configuration-management.repository.id
```
This would need to refer to the new repo

---

updated